pages/api/users/index.ts

```diff
@@ -52,12 +53,16 @@ async function handleGET({ res, userData }: HandlerParams<User[]>) {
       createdAt: true,
       updatedAt: true,
       role: true,
+      isLoggedIn: true,
     },
   });
 
   return res.status(200).json(users);
 }
 
+// CHECK THIS, voiko tämän poistaa / lisätä, että ei käytössä? Tiliä ei voi luoda, koska ei ole kirjautunut sisään
+// Rekistöröityminen tapahtuu /pages/api/auth/register.ts:n kautta
+// /pages/api/auth/register.ts käyttä createUser-funktiota alempana
 async function handlePOST({ req, res }: HandlerParams<User>) {
```

Comment:
// CHECK THIS, voiko tämän poistaa / lisätä, että ei käytössä? Tiliä ei voi luoda, koska ei ole kirjautunut sisään
// Rekistöröityminen tapahtuu /pages/api/auth/register.ts:n kautta
// /pages/api/auth/register.ts käyttä createUser-funktiota alempana

Voisiko tämän POST-requestin poistaa, koska tätä ei voi käyttää ollenkaan. Syynä on se, että tämä request vaatii sisään kirjautumisen, jotta request menee läpi :smile: 

Rekistöröityminen tapahtuu myöskin `/pages/api/auth/register.ts` kautta, joten sinällään tälle POST-requestille ei näyttäisi olevan tarvetta

Reply:
Näkisin, että tämä on admin-endpoint. Ts. normaalit käyttäjät käyttävät `register.ts` tiedostoa, mutta admin voi halutessaan luoda käyttäjiä suoraan tämän kautta esim. testausta varten.

Reply:
Totta, en tullut tuota ajatelleeksi :+1:

Reply:
Tästä puuttuu vielä tarkistus, että käyttäjä on admin eikö vain?